### PR TITLE
Add Configurable Keybind for Toggling Autoplay during Playtesting

### PR DESCRIPTION
**Pull request by QiuFeng54321**
I messed up rebase last commit so I opened a fixed one.

### CODE REVIEW

File: Quaver.Shared/Screens/Options/OptionsMenu.cs [hunk 1]
```diff
@@ -362,6 +362,7 @@ private void CreateSections()
                         new OptionsItemKeybind(containerRect, "Increase Scroll Speed", ConfigManager.KeyIncreaseScrollSpeed),
                         new OptionsItemKeybind(containerRect, "Decrease Map Offset", ConfigManager.KeyDecreaseMapOffset),
                         new OptionsItemKeybind(containerRect, "Increase Map Offset", ConfigManager.KeyIncreaseMapOffset),
+                        new OptionsItemKeybind(containerRect, "Toggle Playtest Autoplay", ConfigManager.KeyTogglePlaytestAutoplay),
```
Comment (Warp9000) [inline Quaver.Shared/Screens/Options/OptionsMenu.cs:365]: This would fit better in the editor section.

**Review by Warp9000 — CHANGES_REQUESTED** [review]

File: Quaver.Shared/Screens/Options/OptionsMenu.cs (hunk 1 above)
Comment (QiuFeng54321) [reply Quaver.Shared/Screens/Options/OptionsMenu.cs:365]: This is also in map preview though

Comment (QiuFeng54321) [reply Quaver.Shared/Screens/Options/OptionsMenu.cs:365]: Oh never mind, it shouldn't be in gameplay anyway

Comment (Warp9000) [reply Quaver.Shared/Screens/Options/OptionsMenu.cs:365]: it can also fit in `song selection` in that case

**Review by Warp9000 — APPROVED** [review]

**Comment by QiuFeng54321** [comment]
a sec, need to resolve conflict

**Pull request merged**